ui/src/containers/Layout.js

```diff
@@ -68,13 +69,38 @@ class Layout extends Component {
       ]
     };
 
+    let currentLanguage = this.props.language === 'fr' ? 'FR' : 'EN';
+
+    const languages = [
+      {
+        label: 'Français',
+        name: 'FR',
+        onClick: () => {
+          currentLanguage = 'FR';
+          this.props.setLang('fr');
+          localStorage.setItem('language', 'fr');
```

Comment:
You should manage ```localStorage``` in the saga ```setLang ``` to avoid a desynchronization between the Redux state and localstorage